### ISSUE TITLE
Using windows Surface pen

### ISSUE THREAD

**Issue by dumbeau** [dumbeau/AutoHotPie#88]
Brought up on Discord by Fox'Say
> The cursor and the pen are kind of separate, but not. When the stylus is on screen, the cursor disappears, but it will reappear after you lift the pen.

> While it recognizes where my stylus is pointed at on screen, It only opens at the mouse cursor

AHP needs to recognize when the pen is being used vs the cursor and open the pie menu accordingly.  Also, pen taps need to be treated like mouse clicks.  This may be related to (#49).

**Comment by dumbeau** [comment]
Windows ink still makes me sad.

**Comment by Da3n0n** [comment]
Yeah windows INK Makes me very sad been banging my head against the wall trying to get all my glorious pies working with my pen but pretty much impossible to use with windows ink and would jsut diable it but i need it to scroll in certain apps like browser :( would love to see this work really would love it . thanks for your amazing application anyway.
